package.json

```diff
@@ -27,6 +27,7 @@
     "@mdi/svg": "^5.3.45",
     "aphrodite": "https://github.com/learningequality/aphrodite/",
     "autosize": "^3.0.21",
+    "consola": "^2.14.0",
```

Comment:
according to our current conventions, this should be in `devDependencies`, not `dependencies

https://github.com/learningequality/kolibri-design-system#adding-dependencies